app/src/main/kotlin/com/wire/android/notification/PendingIntents.kt

```diff
@@ -0,0 +1,122 @@
+package com.wire.android.notification
+
+import android.app.PendingIntent
+import android.content.Context
+import android.content.Intent
+import android.net.Uri
+import androidx.compose.animation.ExperimentalAnimationApi
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material3.ExperimentalMaterial3Api
+import androidx.compose.ui.ExperimentalComposeUiApi
+import com.wire.android.notification.broadcastreceivers.CallNotificationDismissReceiver
+import com.wire.android.notification.broadcastreceivers.MessageNotificationDismissReceiver
+import com.wire.android.notification.broadcastreceivers.NotificationReplyReceiver
+import com.wire.android.ui.WireActivity
+import com.wire.android.util.deeplink.DeepLinkProcessor
+
+//TODO
+fun messagePendingIntent(context: Context, conversationId: String): PendingIntent {
+    return summaryMessagePendingIntent(context)
+}
+
+fun dismissMessagePendingIntent(context: Context, conversationId: String?, userId: String?): PendingIntent {
+    val intent = MessageNotificationDismissReceiver.newIntent(context, conversationId, userId)
+    val requestCode = conversationId?.hashCode() ?: DISMISS_MESSAGE_NOTIFICATION_DEFAULT_REQUEST_CODE
+
+    return PendingIntent.getBroadcast(
+        context.applicationContext,
+        requestCode,
+        intent,
+        PendingIntent.FLAG_IMMUTABLE
+    )
+}
+
+//TODO
+fun callMessagePendingIntent(context: Context, conversationId: String): PendingIntent = summaryMessagePendingIntent(context)
+
+@OptIn(
+    ExperimentalAnimationApi::class,
+    ExperimentalMaterial3Api::class,
+    ExperimentalMaterialApi::class,
+    ExperimentalComposeUiApi::class
+)
```

Comment:
Are these OptIns required for this function? It's not Compose UI, shouldn't have anything to do with the Material or animations 🤔

Reply:
we use `WireActivity` in that function which has these annotations, can't build the app without it here